src/spikeinterface/core/core_tools.py

```diff
@@ -476,3 +477,24 @@ def normal_pdf(x, mu: float = 0.0, sigma: float = 1.0):
     """
 
     return 1 / (sigma * np.sqrt(2 * np.pi)) * np.exp(-((x - mu) ** 2) / (2 * sigma**2))
+
+
+def get_class_info(a_class):
```

Comment:
I strongly suggest against something as vage as `get_class_info` for a function. If I remember well the point of storing this attributes is to store information about how to instantiate the class. From that perspective I think that something like:

* `get_importing_provenance`
* `get_class_origin` 
* `retrieve_class_path` 

or a combination of them. I like `retrieve_importing_provenance` because I think it describes well what it does and it also does not add more suggestions to the get methods that are more used for user relevant information.

Reply:
Totally agree.
THanks you for the feedback. lets go for retrieve_importing_provenance